cmd/eventlistenersink/main.go

```diff
@@ -18,200 +18,53 @@ package main
 
 import (
 	"context"
-	"fmt"
 	"log"
-	"net/http"
-	"strings"
-	"sync"
-	"time"
 
+	"github.com/tektoncd/triggers/pkg/adapter"
 	dynamicClientset "github.com/tektoncd/triggers/pkg/client/dynamic/clientset"
 	"github.com/tektoncd/triggers/pkg/client/dynamic/clientset/tekton"
-	"github.com/tektoncd/triggers/pkg/client/informers/externalversions"
-	triggerLogging "github.com/tektoncd/triggers/pkg/logging"
 	"github.com/tektoncd/triggers/pkg/sink"
-	"github.com/tektoncd/triggers/pkg/system"
-	"go.uber.org/zap"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	cminformer "knative.dev/pkg/configmap/informer"
-	"knative.dev/pkg/controller"
+	evadapter "knative.dev/eventing/pkg/adapter/v2"
 	"knative.dev/pkg/injection"
-	"knative.dev/pkg/injection/sharedmain"
-	"knative.dev/pkg/logging"
-	"knative.dev/pkg/metrics"
-	"knative.dev/pkg/profiling"
+	"knative.dev/pkg/injection/clients/dynamicclient"
 	"knative.dev/pkg/signals"
 )
 
 const (
 	// EventListenerLogKey is the name of the logger for the eventlistener cmd
 	EventListenerLogKey = "eventlistener"
-	// ConfigName is the name of the ConfigMap that the logging config will be stored in
-	ConfigName = "config-logging-triggers"
-)
-
-var (
-	// CacheSyncTimeout is the amount of the time we will wait for the informer cache to sync
-	// before timing out
-	cacheSyncTimeout = 1 * time.Minute
 )
 
 func main() {
-	// set up signals so we handle the first shutdown signal gracefully
 	ctx := signals.NewContext()
 
-	clusterConfig, err := rest.InClusterConfig()
```

Comment:
100% chance something in this red side was wrong. We found this every. single. time. we converted to injection and shared main.

Reply:
You mean, it was wrong and now fixed...?  😅 🤣

Reply:
I noticed at least two issues and fixed at least one (prior to this change), which was some of my motivation for moving to something a bit more common/mature.